frontend/template.go

```diff
@@ -27,73 +30,80 @@ type tmplArguments struct {
-		{{ $option := .URLOption }}
-		{{ $server := .URLServer }}
-		{{ $target := .URLCommand }}
-		{{ if .IsWhois }}
-			{{ $option = "summary" }}
-			{{ $server = .AllServersURL }}
-			{{ $target = "" }}
-		{{ end }}
-		<ul class="navbar-nav mr-auto">
-			<li class="nav-item">
-				<a class="nav-link{{ if .AllServersLinkActive }} active{{ end }}"
-					href="/{{ $option }}/{{ .AllServersURL }}/{{ $target }}"> All Servers </a>
-			</li>
-			{{ range $k, $v := .Servers }}
-			<li class="nav-item">
-				<a class="nav-link{{ if eq $server $v }} active{{ end }}"
-					href="/{{ $option }}/{{ $v }}/{{ $target }}">{{ $v }}</a>
-			</li>
-			{{ end }}
-		</ul>
-		{{ if .IsWhois }}
-			{{ $target = .WhoisTarget }}
-		{{ end }}
-		<form class="form-inline" action="/redir" method="GET">
-			<div class="input-group">
-				<select name="action" class="form-control">
-					{{ range $k, $v := .Options }}
-					<option value="{{ $k }}"{{ if eq $k $.URLOption }} selected{{end}}>{{ $v }}</option>
-					{{ end }}
-				</select>
-				<input name="server" class="d-none" value="{{ $server }}">
-				<input name="target" class="form-control" placeholder="Target" aria-label="Target" value="{{ $target }}">
-				<div class="input-group-append">
-					<button class="btn btn-outline-success" type="submit">&raquo;</button>
-				</div>
-			</div>
-		</form>
-	</div>
-</nav>
-
-<div class="container">
-	{{ .Content }}
-</div>
-
-<script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
-<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.1/dist/js/bootstrap.min.js" integrity="sha256-0IiaoZCI++9oAAvmCb5Y0r93XkuhvJpRalZLffQXLok=" crossorigin="anonymous"></script>
-</body>
-</html>
-`))
+// summary
+
+type SummaryRowData struct {
+	Name        string
+	Proto       string
+	Table       string
+	State       string
+	MappedState string
+	Since       string
+	Info        string
+}
+
+type TemplateSummary struct {
+	ServerName string
+	Raw        string
+	Header     []string
+	Rows       []SummaryRowData
+}
+
+// whois
+type TemplateWhois struct {
+	Target string
+	Result string
+}
+
+// bgpmap
+type TemplateBGPmap struct {
+	Servers []string
+	Target  string
+	Result  string
+}
+
+// bird
+type TemplateBird struct {
+	ServerName string
+	Target     string
+	Result     string
+}
+
+// global variable to hold the templates
+
+var TemplateLibrary map[string]*template.Template
+
+// list of required templates
+
+var requiredTemplates = [...]string{
+	"page",
+	"summary",
+	"whois",
+	"bgpmap",
+	"bird",
+}
+
+// import templates from bindata
+
+func ImportTemplates() {
+
+	// create a new (blank) initial template
+	TemplateLibrary = make(map[string]*template.Template)
+
+	// for each template that is needed
+	for _, tmpl := range requiredTemplates {
+
+		// extract the template definition from the bindata
+		def := MustAssetString("templates/" + tmpl)
```

Comment:
I'm getting `undefined: MustAssetString` with this like. I'm unsure which package I should include to fix this.

Also, it would be better to add extension `.tpl` to the template files, to make them more recognizable.

Reply:
The `undefined: MustAssetString` is sorted by using the right bindata version (detailed now in the Dockerfile). 
Have also updated the template files to use .tpl, that was just laziness from me.